doc/en/source/ngsildbasediotintegration.rst

```diff
@@ -0,0 +1,476 @@
+
+	curl -iX POST 'http://localhost:4041/iot/services' \
+	-H 'fiware-service: openiot' \
+	-H 'fiware-servicepath: /' \
+	-H 'Content-Type: application/json' \
+	--data-raw '{
+    	"services": [
+        	{
+            	"apikey": "4jggokgpepnvsb2uv4s40d59ov",
+            	"cbroker": "http://orion:1026",
+            	"entity_type": "Device",
+            	"resource": "/iot/d",
+            	"attributes": [
+                	{
+                    	"object_id": "bpm", "type": "Property", "name": "heartRate",
+                    	"metadata": { "unitCode": {"type": "Text", "value": "5K" }}
+                	},
+                	{
+                    	"object_id": "s", "name": "status", "type": "Property"
+                	},
+                	{
+                    	"object_id": "gps", "name": "location", "type": "geo:point"
+                	}
+            	],
+            	"static_attributes": [
+                	{
+                    	"name": "category", "type": "Property", "value": "sensor"
+                	},
+                	{
+                    "name": "supportedProtocol", "type": "Property", "value": "ul20"
+                	}
+            	]
+        	}
+    	   ]
+	}'
+
+
+**Note:** To know about Fiware-Service and Fiware-ServicePath please click  `here`_
+
+.. _`here`: https://ngsi-ld-tutorials.readthedocs.io/en/latest/iot-agent.html#connecting-iot-devices
+
+
+In the example the IoT Agent is informed that the `/iot/d` endpoint will be used and that devices will authenticate
+themselves by including the token **4jggokgpepnvsb2uv4s40d59ov**. For an UltraLight IoT Agent this means devices will be
+sending GET or POST requests to:**http://iot-agent:7896/iot/d?i=<device_id>&k=4jggokgpepnvsb2uv4s40d59ov**
+
+**Step2** Provisioning an Actuator
+----------------------------------
+The example below provisions a water with the `deviceId=water001`. The endpoint is
+`http://iot-sensors:3001/iot/water001` and it can accept the `on` command. The `transport=HTTP` attribute defines the
+communications protocol to be used.
+
+.. code-block:: console  
+
+	curl -L -X POST 'http://localhost:4041/iot/devices' \
+    	-H 'fiware-service: openiot' \
+    	-H 'fiware-servicepath: /' \
+    	-H 'Content-Type: application/json' \
+	--data-raw '
+		{
+	"devices": [{
+		"device_id": "water001",
+		"entity_name": "urn:ngsi-ld:Device:water001",
+		"entity_type": "Device",
+		"protocol": "PDI-IoTA-UltraLight",
+		"transport": "HTTP",
+		"endpoint": "http://context-provider:3001/iot/water001",
+		"commands": [{
+				"name": "on",
+				"type": "command"
+			},
+			{
+				"name": "off",
+				"type": "command"
+			}
+		],
+		"static_attributes": [{
+			"name": "controlledAsset",
+			"type": "Relationship",
+			"value": "urn:ngsi-ld:Building:barn001"
+		}]
+	    }]
+	}
+	'
+
+
+**Step3** To see the state of the water sprinkler change through device monitor URL:**localhost:3000/device/monitor** send the below PATCH request directly to the IoT Agent's North Port
```

Comment:
i hope it is Northbound port instead of North port.

Reply:
It is North port